packages/cli/src/lib/webpack/transform-config.js

```diff
@@ -239,11 +239,13 @@ class WebpackConfigHelpers {
 					: [{ rule, ruleIndex, loader: loaders, loaderIndex: -1 }]
 			)
 			.reduce((arr, loaders) => arr.concat(loaders), [])
-			.filter(
-				({ loader }) =>
-					(typeof loader === 'string' && loader.includes(name)) ||
-					(typeof loader.loader === 'string' && loader.loader.includes(name))
-			);
+			.filter(({ loader }) => {
+				if (typeof loader === 'string') return loader.includes(name);
+				return typeof loader.loader === 'string' &&
```

Comment:
Not 100% that we need this type check here, but Webpack's types are quite loose and I don't want to break anyone's setup by changing behavior.